service/service.go

```diff
@@ -24,6 +24,7 @@ type Port struct {
 	Type        string
 	Port        int64
 	ServicePort int64
+	IP          string
```

Comment:
Small design question: why not use the `[net.IP](https://golang.org/pkg/net/#IP)` type to store IPs?